### DISCUSSION TITLE
Does this work on Mac

### DISCUSSION THREAD

**Discussion by StockMilkMan** [LivingFray/HaloCEVR#50] [Q&A]
I apologize for asking this, but I'm new to the whole mod thing since I'm used to being on a console. I have an iMac running Sonoma and didn't know if I was able to download a copy of CE for Mac and then use the VR mod with it. Any help with this would be greatly appreciated!

**Comment by Daryl-EndicottDavies** [comment]
the base game is for windows.  I guess if you run something like parallels?  But what headset?
